frontend/src/pages/live_routes.tsx

```diff
@@ -10,6 +10,12 @@ const DynamicLiveRoutesComponent = dynamic(() =>
 export default function LiveRoutesPage() {
     const {data: session} = useSession();
 
+    /* TODO
```

Comment:
This TODO is in the wrong file (and already solved) I think. My-routes are redirected to your component. Live-routes are for admin and superstudents. My-routes are for students (which will use your component to do their work).